app/javascript/components/TopToolBar.jsx

```diff
@@ -12,10 +12,16 @@ export default class TopToolBar extends Component {
     this.handleSave = this.handleSave.bind(this);
   }
 
+  //---------------------------
+  //Zapis zagrywki
```

Comment:
comments should be written in English. Also remove all those decorations like `//---------------------------`. I think those comments can be skipped at all by having better function names